x-pack/test/security_solution_api_integration/test_suites/entity_analytics/utils/risk_engine.ts

```diff
@@ -502,6 +503,14 @@ export const riskEngineRouteHelpersFactory = (
       .set(X_ELASTIC_INTERNAL_ORIGIN_REQUEST, 'kibana')
       .send()
       .expect(200),
+  privilegesForUser: async ({ username, password }: { username: string; password: string }) =>
```

Comment:
Nit: since this route requires its own authentication and doesn't rely on the default, it's quite different from the other routes in this collection; I would consider making this its own standalone helper, mainly just to be "least surprising."